### PR TITLE
Adding support the i18n-tasks gem

### PR DESCRIPTION
**Pull request by nickrobertson**
This will allow people to verify that they are not missing any rails translations using the i18n-tasks gem.

See the following: https://github.com/glebm/i18n-tasks

This will add a new command to lois called `lois i18n-tasks`.
It will take an optional argument `-l` which will take an i18n.locale to only check the specified locale.
For instance you could run: `lois i18n-tasks -l en-US`.
Not passing that option will check all available locales in your project.
Internally this runs `i18n-tasks health`.

### CODE REVIEW

**Comment by ketiko** [comment]
I don't want to confuse the I18n-task files from what's actually lois code.  Having the `config/locales` and `app` folders I think could lead to confusion.

Could you move `app` to `spec/dummy_app`?  And `config/locales` to `spec/locales`?

Then update `config/i18n-tasks.yml` to use the new locations.

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-       - image: circleci/ruby:2.4.1-node-browsers
-         environment:
-           COVERAGE: true
+      - image: circleci/ruby:2.4.1-node-browsers
```
Comment (ketiko) [inline .circleci/config.yml:10]: Thanks for fixing that formatting!

File: lib/lois/ci/travis.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@ def repository
 
       def commit_sha
         sha = ENV.fetch('TRAVIS_PULL_REQUEST_SHA')
-        sha.length == 0 ? ENV.fetch('TRAVIS_COMMIT') : sha
+        sha.empty? ? ENV.fetch('TRAVIS_COMMIT') : sha
```
Comment (ketiko) [inline lib/lois/ci/travis.rb:14]: I presume this is a rubocop fix?

File: lois.gemspec [hunk 3]
```diff
@@ -1,4 +1,4 @@
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path('lib', __dir__)
```
Comment (ketiko) [inline lois.gemspec:1]: Is this another rubocop fix?

File: .circleci/config.yml [hunk 4]
```diff
@@ -39,6 +39,11 @@ jobs:
           key: v2-dependencies-{{ checksum "lois.gemspec" }}
 
       # run tests!
+      - run:
```
Comment (ketiko) [inline .circleci/config.yml]: This block needs to move to after we install the gem with `bin/rake install`.  Can you move this block after the `Reek` one and before the tests instead?

**Comment by ketiko** [comment]
This is a great addition.  Thank you so much for taking time to contribute!

File: lib/lois/ci/travis.rb (hunk 2 above)
Comment (nickrobertson) [reply lib/lois/ci/travis.rb:14]: That is correct, rubocop recommended this fix.

File: lois.gemspec (hunk 3 above)
Comment (nickrobertson) [reply lois.gemspec:1]: That is correct, rubocop recommended this fix.

**Comment by ketiko** [comment]
Sorry I haven't gotten back to this, let me see if I can figure out why the build isn't working.

**Comment by ketiko** [comment]
@nickrobertson Could you do a version bump in this PR as well?

**Review by ketiko — APPROVED** [review]

**Review by ketiko — APPROVED** [review]

**Pull request merged**